AppFramework/Action/GREYActions.m

```diff
@@ -573,24 +573,23 @@ + (void)initialize {
             }
             // Wait for keyboard to show up and any other UI changes to take effect.
             if (![GREYKeyboard waitForKeyboardToAppear]) {
-              NSString *description =
-                  @"Keyboard did not appear after tapping on [Element]. "
-                  @"Are you sure that tapping on this element will bring up the keyboard?";
               __block NSString *elementDescription;
               grey_dispatch_sync_on_main_thread(^{
                 elementDescription = [element grey_description];
               });
-              NSDictionary<NSString *, NSString *> *glossary = @{@"Element" : elementDescription};
-              I_GREYPopulateErrorNoted(errorOrNil, kGREYInteractionErrorDomain,
-                                       kGREYInteractionActionFailedErrorCode, description,
-                                       glossary);
+              NSString *description = [NSString stringWithFormat:
+                                       @"Keyboard did not appear after tapping on an element."
+                                       @"Are you sure that tapping on this element will bring up the keyboard?"
+                                       @"\nElement: %@", element];
```

Comment:
Instead of newline for each items of the glossary, how about adding numbering, or bullet points? I feel like that would be easier and cleaner than adding spaces, as well as parsing (if we have to). What do you think?